packages/src/rpm-systemd/post.sh

```diff
@@ -8,7 +8,7 @@ if ! getent passwd pganalyze > /dev/null; then
 fi
 
 if ! getent group pganalyze > /dev/null; then
-  addgroup --system --quiet pganalyze
+  groupadd --system pganalyze
```

Comment:
`--quiet` is not an option for `groupadd`, but the command produces no output anyway, so the flag is moot.